drivers/pwm/pwm_nrfx.c

```diff
@@ -339,10 +339,15 @@ static int pwm_nrfx_pm_action(const struct device *dev,
 
 #define PWM(dev_idx) DT_NODELABEL(pwm##dev_idx)
 #define PWM_PROP(dev_idx, prop) DT_PROP(PWM(dev_idx), prop)
+#define PWM_HAS_PROP(idx, prop) DT_NODE_HAS_PROP(PWM(idx), prop)
 
 #define PWM_NRFX_DEVICE(idx)						      \
 	NRF_DT_CHECK_NODE_HAS_PINCTRL_SLEEP(PWM(idx));			      \
-	static struct pwm_nrfx_data pwm_nrfx_##idx##_data;		      \
+	static uint16_t pwm_##idx##_seq_values				      \
+			[NRF_PWM_CHANNEL_COUNT]				      \
```

Comment:
nit: I think it will be more readable when this is in a single line and only `PWM_MEMORY_SECTION` separately.

Reply:
fixed